sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/resourcemanager/AzureEventHubsResourceManagerAutoConfiguration.java

```diff
@@ -18,7 +18,7 @@
 import org.springframework.core.annotation.Order;
 
 /**
- *
+ * An auto-configuration for EventHubs
```

Comment:
A good example for this javadoc is https://github.com/spring-projects/spring-boot/blob/2b7a034776f567946299083bbb9eed6ef9dadc5e/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/elasticsearch/ElasticsearchRestClientAutoConfiguration.java#L31, but we should make it clear here that it's not the autoconfiguration for Event Hubs, it's the autoconfiguration for Event Hubs resource manager.